src/interfaces/standings.interface.ts

```diff
@@ -0,0 +1,51 @@
+  import {Problem} from './problem.interface'
+  import {Member} from './member.interface'
+  
+
+  export interface Standings {
+    contest: Contest
+    problems: Problem[]
+    rows: Row[]
+  }
+  
+  export interface Contest {
+    id: number
+    name: string
+    type: string
+    phase: string
+    frozen: boolean
+    durationSeconds: number
+    startTimeSeconds: number
+    relativeTimeSeconds: number
+  }
+  
+  
+  export interface Row {
```

Comment:
The API is referencing to this type as `RanklistRow` I would suggest renaming this to avoid any differences.

You are also missing `lastSubmissionTimeSeconds` optional field.